sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -242,7 +242,10 @@ Query the Azure Digital Twins instance for digital twins using the [Azure Digita
 ```C# Snippet:DigitalTwinsSampleQueryTwins
 // This code snippet demonstrates the simplest way to iterate over the digital twin results, where paging
 // happens under the covers.
-AsyncPageable<BasicDigitalTwin> asyncPageableResponse = client.QueryAsync<BasicDigitalTwin>("SELECT * FROM digitaltwins");
+AsyncPageable<BasicDigitalTwin> asyncPageableResponse = client.QueryAsync<BasicDigitalTwin>(
```

Comment:
You shouldn't manually change the code snippet, you have to add another snippet section in the sample code and reference it here. 
also, we would like to add the method signature with AdtQueryBuilder as the input parameter and not replace the old one with it. users should see both.